src/frontend/src/app/header/settings/options/switch-without-saving-option.ts

```diff
@@ -0,0 +1,5 @@
+export enum SwitchWithoutSavingOption {
```

Comment:
De file type enums zijn in hoofdletters, zullen we als styleguide nemen dat enums in caps zijn?

Reply:
I thought that was because they are abbreviations. ModeType is CamelCase...

Reply:
https://www.typescriptlang.org/docs/handbook/enums.html

https://stackoverflow.com/questions/49091786/how-to-use-an-enum-in-an-angular-component